lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -28,6 +33,10 @@ def activate_shopify_session
       end
     end
 
+    def user_session_expected?
```

Comment:
minor: can we make this a private method?

Reply:
See latest changes!